doc/src/modules/stats.rst

```diff
@@ -3,6 +3,10 @@ Stats
 
 .. automodule:: sympy.stats
 
+.. automodule:: sympy.stats.drv_types
+.. automodule:: sympy.stats.frv_types
+.. automodule:: sympy.stats.crv_types
```

Comment:
I think you should remove lines 6, 7, 8. Also, you should make a separate branch while working on an issue.

Reply:
The AttributeError is occuring because of lines 6,7,8. Is there a way I can work around that and still be able to include those files docstrings ?

Reply:
They are already included. These lines don't have any meaning

Reply:
Issue #16494 wanted the list of functions in those files to be listed out in stats.rst, initially I manually listed them out but was suggested to do so using the module's docstring which already contains the listing ( PR #16497 ).


> They are already included. These lines don't have any meaning

Sorry I'm a bit new to Sphinx, how are these lists already included in the .rst file ?

Reply:
Either way I couldn't think of a way to keep those lines without having an AttributeError. Removed them for now.